### ISSUE TITLE
Public Search of Tags

### ISSUE THREAD

**Issue by Breeshee** [scholarslab/Neatline#294]
Hello!

In the backend, we are able to make use of the tags in Styles to retrieve tagged records.

However, we wish to make a thematic search available in the public view. Is there a way for the records to be retrievable by searching with a tag in the public search bar?

Thanks, 

Bria

**Comment by waynegraham** [comment]
@Breeshee No, the "tags" are not exposed to the Omeka Search API. We may need to take a look at what Neatline information should be exposed to the Omeka basic search.
